packages/core/src/bodl/bodl-emitter-service.ts

```diff
@@ -96,8 +102,10 @@ export default class BodlEmitterService implements BodlService {
     private getProducts(lineItems: LineItemMap, currencyCode: string): BODLProduct[] {
         const customItems: BODLProduct[] = (lineItems.customItems || []).map(item => ({
             product_id: item.id,
-            product_sku: item.sku,
-            price: item.listPrice,
+            sku: item.sku,
+            base_price: item.listPrice,
+            sale_price: item.listPrice,
+            purchase_price: item.listPrice,
```

Comment:
May need to check with other members if `listPrice` is the best field for this 🤔

Reply:
I see you have a differnt logic on line 146

Reply:
@animesh1987 per doc custom item has only listPrice available, I'm quite not sure if other fields make impact or used for this case